src/components/Form/Form.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todos';
+import { DefaultErrorsState, User } from '../../types';
+import { TextField } from '../TextField/TextField';
+import { getRandomDigits } from '../../utils';
+import { SelectUser } from '../SelectUser';
+import todos from '../../api/todos';
+
+type Props = {
+  users: User[];
+  addTodo: (newTodo: Todo) => void;
+};
+
+type DefaultState = {
+  titleInput: string;
+  userSelect: string;
+};
+
+const defaultState: DefaultState = {
+  titleInput: '',
+  userSelect: '0',
+};
+
+const defaultErrorState: DefaultErrorsState = {
+  titleInput: '',
+  userSelect: '',
+};
+
+const inputId = `${getRandomDigits()}`;
```

Comment:
Ensure that `inputId` is unique for each form instance to prevent potential conflicts, especially if multiple forms are rendered simultaneously.